mail_tracking/controllers/main.py

```diff
@@ -48,40 +40,52 @@ def _request_metadata(self):
             'ua_family': request.user_agent.browser or False,
         }
 
-    def _tracking_open(self, env, tracking_id, event_type, **kw):
-        tracking_email = env['mail.tracking.email'].search([
-            ('id', '=', tracking_id),
-        ])
-        if tracking_email:
-            metadata = self._request_metadata()
-            tracking_email.event_create('open', metadata)
-        else:
-            _logger.warning(
-                "MailTracking email '%s' not found", tracking_id)
-
-    def _tracking_event(self, env, tracking_id, event_type, **kw):
-        metadata = self._request_metadata()
-        return env['mail.tracking.email'].event_process(
-            http.request, kw, metadata, event_type=event_type)
-
-    @http.route('/mail/tracking/all/<string:db>',
+    @http.route(['/mail/tracking/all/<string:db>',
+                 '/mail/tracking/event/<string:db>/<string:event_type>'],
                 type='http', auth='none', csrf=False)
```

Comment:
This is implicit in GET requests, right?

```suggestion
                 type='http', auth='none')
```

Reply:
For me, this route doesn't make sense here... this will be used by mail_tracking_mailgun.

Mailgun documentation says that uses HTTP POST and a specific error reply '406':
https://documentation.mailgun.com/en/latest/user_manual.html#webhooks

Now, the controller in this module (mail_tracking) doesn't have any "methods" defined. This allow to be called using POST (as mailgun does) and for this case you need have defined "csrf" to false.
As said, this module doesn't send the specific error code when mailgun is calling.

Reply:
@Tardo we are still using old Mailgun webhooks, so this is why you find this "incoherence".

Reply:
OK, I guess that when we update `mail_tracking_mailgun` to use the new hooks we can tackle this.